__tests__/app.test.js

```diff
@@ -236,3 +261,151 @@ describe("GET /api/articles/:article_id/comments", () => {
 			});
 	});
 });
+describe("POST /api/articles/:article_id/comments", () => {
+	it("should send the correct status code back (201) when passed the correct comment object ", () => {
+		return request(app)
+			.post("/api/articles/4/comments")
+			.send({
+				username: "lurker",
+				body: "test comment",
+			})
+			.expect(201);
+	});
+	it("should send the comment object back if the correct things are contained within the post request", () => {
+		return request(app)
+			.post("/api/articles/4/comments")
+			.send({
+				username: "lurker",
+				body: "test comment",
+			})
+			.then((data) => {
+				expect(typeof data.body === "object").toBeTrue();
+				expect(Object.keys(data.body).length).toBe(6);
+			});
+	});
+	it("should send the correct status code back (400) when passed nothing", () => {
+		return request(app).post("/api/articles/4/comments").expect(400);
+	});
+	it("should send the correct status code back (400) when passed an incomplete comment object - with the body ommited", () => {
+		return request(app)
+			.post("/api/articles/4/comments")
+			.send({
+				username: "lurker",
+			})
+			.expect(400)
+			.then((data) => {
+				expect(data.body.msg).toBe("Bad request!");
+			});
+	});
+	it("should send the correct status code back (400) when passed an imcomplete comment object - with the username ommited", () => {
+		return request(app)
+			.post("/api/articles/4/comments")
+			.send({
+				body: "test comment",
+			})
+			.expect(400)
+			.then((data) => {
+				expect(data.body.msg).toBe("Bad request!");
+			});
+	});
+	it("should send the correct status code back (400) when passed a comment object with too many key value pairs", () => {
+		return request(app)
+			.post("/api/articles/4/comments")
+			.send({
+				username: "lurker",
+				body: "test comment",
+				info: "please cause an error :)",
+			})
+			.expect(400)
+			.then((data) => {
+				expect(data.body.msg).toBe("Bad request!");
+			});
+	});
+	it("should send the correct status code back (404) when trying to post a comment to an article that doesnt exist ", () => {
+		return request(app)
+			.post("/api/articles/440/comments")
+			.send({
+				username: "lurker",
+				body: "test comment",
+			})
+			.expect(404)
+			.then((data) => {
+				expect(data.body.msg).toBe("Not found!");
+			});
+	});
+	it("should send the correct status code back (400) when passed a comment object that contains one incorrect keys", () => {
```

Comment:
this test case and the following is covered by your incomplete comment object tests :)